ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -1120,7 +1120,7 @@ enum pkcs11_rc check_created_attrs_against_token(struct pkcs11_session *session,
 	/*
 	 * TODO: START_DATE and END_DATE: complies with current time?
 	 */
-	return PKCS11_CKR_OK;
+	return check_access_attrs_against_token(session, head);
```

Comment:
This breaks the usage pattern that has been used before.

And now there is double call to `check_access_attrs_against_token()` with: `entry_create_object()`.

Also keypair generation has the same pattern.

I would drop this commit and add it in unwrapping code as used in: `entry_create_object()`.

Reply:
Thanks, i had missed that. You are right, I will add the additional check in unwrap/derive rather than changing check_access_attrs_against_token()